detox/src/devices/DriverRegistry.js

```diff
@@ -29,6 +29,7 @@ DriverRegistry.default = new DriverRegistry({
   'ios.simulator': require('./drivers/ios/SimulatorDriver'),
   'android.emulator': require('./drivers/android/emulator/EmulatorDriver'),
   'android.attached': require('./drivers/android/attached/AttachedAndroidDriver'),
+  'android.genycloud': require('./drivers/android/genycloud/GenyCloudDriver'),
```

Comment:
Don't consider me too nitpicking, but apparently, the official name for this is not `GenyCloud`, but `GenymotionCloud`, so a better name would be a self-documenting name which is `GenymotionCloudDriver`, overall.